README.md

```diff
@@ -26,7 +26,7 @@ git clone https://github.com/Zuhaib042/portfolio.git
 
 ### Prerequisites
 
-Basic Knowledge of HTML and CSS
```

Comment:
``` ## Authors ```

- [ ] Update your Readme, This is a group project. Please let's include all the authors in the README file. This makes the README file professional and descriptive. 👨🏻‍🎨 

- The README file is the first file a person will see when they encounter your project, so it should be fairly brief but detailed. It will make your project stand out from a bunch of others.